src/ovp_pipeline/source_signals/author_rules.py

```
@@ -0,0 +1,150 @@
+                    norm = (alias or "").lower().lstrip("@").strip()
+                    if norm:
+                        index[norm] = record
+        self._index = index
+        return index
+
+    def _extract_handle_from_url(self, source_url: str) -> str | None:
+        try:
+            parsed = urlparse(source_url)
+        except ValueError:
+            return None
+        host = (parsed.netloc or "").lower().lstrip("www.")
```

Ruff (0.15.12)

[warning] Using `.strip()` with multi-character strings is misleading

(B005)